File: src/main/java/com/senderman/miniroulette/model/Player.java
package com.senderman.miniroulette.model;

public class Player {

    private final long id;
    private final String name;

    public Player(long id, String name) {
        this.id = id;
        this.name = name;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: src/main/kotlin/com/senderman/miniroulette/gameobjects/Player.kt
package com.senderman.miniroulette.gameobjects

import com.senderman.neblib.TgUser

class Player(id: Int, name: String, var coins:Int = 0): TgUser(id, name), Comparable<Player>{
    val bets = ArrayList<Bet>()
    override fun compareTo(other: Player): Int = this.coins.compareTo(other.coins)
}